src/xcsync/Resources/Strings.resx

```diff
@@ -276,9 +284,24 @@
 
 	<!-- Watch Messages -->
 	<data name="Watch.HeaderInformation" xml:space="preserve">
-		<value>Syncing changes between project '{0}' and '{1}'.</value>
-		<comment>{0} is the path to the project file, {1} is the path to the target project</comment>
+		<value>Continuously syncing files between '{0}' and '{1}' for '{2}' platform.</value>
+		<comment>{0} is the path to the project file, {1} is the path to the target project, {2} is the target platform moniker</comment>
+	</data>
+	<data name="Watch.ReceivedCtrlC" xml:space="preserve">
+		<value>Received CTRL-C, exiting.</value>
```

Comment:
Capitalization of `CTRL` should be consistent with the string in line 303. I don't think it matters which way, as long as they match.